packages/editor/src/editor/Editor.tsx

```diff
@@ -69,6 +70,7 @@ const Editor: React.FC<EditorProps> = ({
       cellPlugins={cellPlugins}
       lang={lang}
       cellSpacing={cellSpacing}
+      components={components}
```

Comment:
we should not pass `components` here as is.

we have to separate components that are solely used for the editor-ui and components that are responsible for displaying the content (no matter of read-only or not)

Reply:
@macrozone it's hidden inside the implementation component and there is a good change "components" are memoized already, so it didn't feel right to mess with that. But I'm okay extracting those if you insist. Do you want to split those at this level or expose them in a different way to the clients?